libs/components/lists/src/lib/modules/filter/fixtures/filter-button.component.fixture.ts

```diff
@@ -9,10 +9,10 @@ export class FilterButtonTestComponent {
   public filtersActive = false;
   public showButtonText = false;
   public buttonClicked = false;
-  public buttonId: string;
-  public ariaExpanded: boolean;
-  public ariaControls: string;
-  public skyThemeSettings: SkyThemeSettings;
+  public buttonId: string | undefined;
+  public ariaExpanded: boolean | undefined;
+  public ariaControls: string | undefined;
+  public skyThemeSettings!: SkyThemeSettings;
```

Comment:
could we set a default for this so we can avoid using `!`?

Reply:
I did this because Angular doesn't see it as being set in the constructor (due to the function call) but it will always have a value. I could have copied the code out of `useDefaultTheme()` so it wouldn't complain, but I think this is exactly the situation that `!` is intended to be used for.